task.go

```diff
@@ -64,7 +64,9 @@ func (e *Executor) Run(ctx context.Context, calls ...taskfile.Call) error {
 	for _, c := range calls {
 		if _, ok := e.Taskfile.Tasks[c.Task]; !ok {
 			// FIXME: move to the main package
-			e.PrintTasksHelp()
+			// FIXME: (ard.kevin.84@gmail.com) changed the PrintTasksHelp signature to support show all/some.
+			// 	False preserves original behavior, but should be reviewed.
+			e.PrintTasksHelp(false)
```

Comment:
The note is not needed IMO. Let's keep the default behavior which is to list documented tasks only.

```suggestion
			e. ListTasksWithDesc()
```

Reply:
resolved in https://github.com/therealkevinard/task/commit/42702e81b35c5b4960c94ff65577f6c4314b0ccd